portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSingleSelectionField.testcase

```diff
@@ -83,13 +83,66 @@ definition {
 	}
 
 	@description = "Verify that a Single Selection Field can be set to repeatable"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanBeSetToRepeatable {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-132194 CanBeSetToRepeatable pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Single Selection");
+
+		FormFields.editOptionsRow(
+			optionValue = "Option A",
+			rowNumber = "1");
+
+		FormFields.editOptionsRow(
+			optionValue = "Option B",
+			rowNumber = "2");
+
+		Navigator.gotoNavTab(navTab = "Advanced");
+
+		Check.checkToggleSwitch(
+			key_toggleSwitchLabel = "Repeatable",
+			locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+
+		Click(locator1 = "Button#SAVE");
+
+		Button.clickPublish();
+
+		FormsAdminNavigator.gotoPublishedForm();
+
+		Click(locator1 = "FormFields#FIELD_ADD_DUPLICATE");
+
+		AssertElementPresent(
+			key_rowNumber = "2",
+			locator1 = "FormFields#SELECT_SINGLE_SELECTION_ROW");
```

Comment:
You can use FormFields.addRepeatableField, as it is going to call a macro that has the same effect.
An example can be seen [here](https://github.com/liferay/liferay-portal/blob/fdcbefc43088ad9bb8a05e51c9b5d22dbf507589/portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase#L4590-L4592), you'll need to send the field name and number of times that you want to repeat the field.